scenario/scenario_shacl.ttl

```diff
@@ -278,4 +198,78 @@ scenario:TrafficShape a sh:NodeShape ;
+    sh:targetClass scenario:Content .
+
+scenario:QuantityShape a sh:NodeShape ;
+    sh:closed false ;
+    sh:ignoredProperties ( rdf:type ) ;
+    sh:property [ skos:example "2" ;
+            sh:datatype xsd:unsignedInt ;
+            sh:description "Count of traffic temporary objects"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of temporaryTrafficObjects failed!"@en ;
+            sh:name "temporaryTrafficObjects"@en ;
+            sh:order 1 ;
+            sh:path scenario:temporaryTrafficObjects ],
+        [ skos:example "10" ;
+            sh:datatype xsd:unsignedInt ;
+            sh:description "Count of traffic objects"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of numberTrafficObjects failed!"@en ;
+            sh:name "numberTrafficObjects"@en ;
+            sh:order 0 ;
+            sh:path scenario:numberTrafficObjects ],
+        [ skos:example "myDriver, BestPedestrianController2000" ;
+            sh:datatype xsd:string ;
+            sh:description "Usage of simulation core internal or external controllers."@en ;
+            sh:message "Validation of controllers failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "controllers"@en ;
+            sh:order 5 ;
+            sh:path scenario:controllers ],
+        [ skos:example "8" ;
+            sh:datatype xsd:unsignedInt ;
+            sh:description "Count of traffic permanent objects"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of permanentTrafficObjects failed!"@en ;
+            sh:name "permanentTrafficObjects"@en ;
+            sh:order 2 ;
+            sh:path scenario:permanentTrafficObjects ] ;
+    sh:targetClass scenario:Quantity .
+
+scenario:QualityShape a sh:NodeShape ;
+    sh:closed false ;
+    sh:ignoredProperties ( rdf:type ) ;
+    sh:property [ skos:example "0.1" ;
+            sh:datatype xsd:float ;
+            sh:description "Accuracy of moving objects (only applicable if scenario is based on real measurments)."@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of accuracyObjects failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "accuracyObjects"@en ;
+            sh:order 4 ;
+            sh:path scenario:accuracyObjects ],
+        [ skos:example "Calibration performed at origin using system XYZ" ;
+            sh:datatype xsd:string ;
+            sh:description "Description of any calibration steps performed prior to measurement"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of calibration failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "calibration"@en ;
+            sh:order 5 ;
+            sh:path scenario:calibration ] ;
+    sh:targetClass scenario:Quality .
+
+scenario:DataSourceShape a sh:NodeShape ;
+    sh:closed false ;
+    sh:ignoredProperties ( rdf:type ) ;
+    sh:property [ skos:example "captured reality, synthetic" ;
+            sh:datatype xsd:string ;
```

Comment:
Ist hier ein `sh:in` möglich?

Reply:
Hmm... ich könnte hier auf jeden Fall ein `sh:in` vorschreiben. Allerdings ist es fraglich ob es vollumfänglich ist. Habe mal ein Versuch integriert. Aber ich musste dafür dann ein weiteres Attribut hinzufügen, welche den Typ nochmal beschreibt, das muss aber ein Freitext sein.

Reply:
Passt das so, dann würde ich den Konflikt resolven?

Reply:
Verstanden. 
Was ich mir auch vorstellen kann, wäre ein zusätzliches "other" in dem `sh:in`. 
Das überlasse ich Dir, ob das fachlich hilft :)
Bei der `sh:description` in `sourceDescription` würde ich ggfs schreiben, dass das ein Freitext ist, welcher den `sh:in` oben präzisiert.
Aber auch das überlasse ich Dir, ob das fachlich Sinn macht.

Reply:
Das wär natürlich eine Option (mit dem other), ich würd's aktuell mal drauf ankommen lassen und warten bis tatsächlich einer noch eine andere Option benötigt.
Der Hinweis bzgl. der Detaillierung ist gut, habe ich mit aufgenommen.